appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/MainView.java

```diff
@@ -30,6 +33,10 @@
  * Results tab content
  */
 public class MainView extends AbstractAppSecContent {
```

Comment:
To make the logger name meaningful, I suggest to rename this class to `AppSecView` as we have in the V24 module.

Reply:
Good point. Renamed.